receipt/build.gradle

```diff
@@ -2,6 +2,16 @@ apply from: "$rootProject.projectDir/android-library.gradle"
 
 dependencies {
     api project(":common")
+
+    implementation "com.google.android.material:material:1.0.0"
```

Comment:
are these implementation dependencies temporary? If not I think we should add the version as variable. If I'm not mistaken @azakrevska-epam did that in another pr